permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
+
+# OH GOD WHAT IS ANYTHING FOR?
+
+################################################################################
+
+packages <- c("gtools", "readr", "tibble", "dplyr")
+
+lapply(packages, library, character.only = TRUE)
+
+
 
 x <- c("A","B","C","D","E","F")
```

Comment:
I didn't change this, but I suggest following a general principle of naming objects succinctly but descriptively. Try to avoid things like `x` or `df` if you can give the objects a short name that is easy to remember and to identify.